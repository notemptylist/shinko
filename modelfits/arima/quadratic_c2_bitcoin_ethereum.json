{
    "numlags": 400,
    "results": [
        {
            "mean": 0.016536318802294416,
            "order": [
                1,
                2,
                1
            ],
            "rmse": 13.073837486779503,
            "tstamp": 1639950202.5032058
        },
        {
            "mean": 0.016536318802294416,
            "order": [
                3,
                2,
                0
            ],
            "rmse": 13.118096316970318,
            "tstamp": 1639950202.5032058
        },
        {
            "mean": 0.016536318802294416,
            "order": [
                1,
                2,
                0
            ],
            "rmse": 13.14593045742272,
            "tstamp": 1639950202.5032058
        }
    ],
    "stream": "quadratic_c2_bitcoin_ethereum.json",
    "todo": [
        [
            0,
            0,
            0
        ],
        [
            0,
            0,
            1
        ],
        [
            0,
            0,
            2
        ],
        [
            0,
            1,
            0
        ],
        [
            0,
            1,
            1
        ],
        [
            0,
            1,
            2
        ],
        [
            0,
            2,
            0
        ],
        [
            0,
            2,
            1
        ],
        [
            0,
            2,
            2
        ],
        [
            1,
            0,
            0
        ],
        [
            1,
            0,
            1
        ],
        [
            1,
            0,
            2
        ],
        [
            1,
            1,
            0
        ],
        [
            1,
            1,
            1
        ],
        [
            1,
            1,
            2
        ],
        [
            1,
            2,
            2
        ],
        [
            2,
            0,
            0
        ],
        [
            2,
            0,
            1
        ],
        [
            2,
            0,
            2
        ],
        [
            2,
            1,
            0
        ],
        [
            2,
            1,
            1
        ],
        [
            2,
            1,
            2
        ],
        [
            2,
            2,
            0
        ],
        [
            2,
            2,
            1
        ],
        [
            2,
            2,
            2
        ],
        [
            3,
            0,
            0
        ],
        [
            3,
            0,
            1
        ],
        [
            3,
            0,
            2
        ],
        [
            3,
            1,
            0
        ],
        [
            3,
            1,
            1
        ],
        [
            3,
            1,
            2
        ],
        [
            3,
            2,
            1
        ],
        [
            3,
            2,
            2
        ],
        [
            6,
            0,
            0
        ],
        [
            6,
            0,
            1
        ],
        [
            6,
            0,
            2
        ],
        [
            6,
            1,
            0
        ],
        [
            6,
            1,
            1
        ],
        [
            6,
            1,
            2
        ],
        [
            6,
            2,
            0
        ],
        [
            6,
            2,
            1
        ],
        [
            6,
            2,
            2
        ]
    ],
    "version": "0.1"
}