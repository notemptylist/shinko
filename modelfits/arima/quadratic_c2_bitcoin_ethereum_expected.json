{
    "numlags": 330,
    "results": [
        {
            "mean": 0.2545980199856596,
            "order": [
                6,
                0,
                1
            ],
            "rmse": 2.9505155548535615,
            "tstamp": 1639794624.5921724
        },
        {
            "mean": 0.2545980199856596,
            "order": [
                3,
                1,
                2
            ],
            "rmse": 3.2305039973589618,
            "tstamp": 1639794624.5921724
        },
        {
            "mean": 0.2545980199856596,
            "order": [
                0,
                2,
                0
            ],
            "rmse": 7.472681193068428,
            "tstamp": 1639794624.5921724
        }
    ],
    "stream": "quadratic_c2_bitcoin_ethereum_expected.json",
    "todo": [
        [
            0,
            0,
            0
        ],
        [
            0,
            0,
            1
        ],
        [
            0,
            0,
            2
        ],
        [
            0,
            1,
            0
        ],
        [
            0,
            1,
            1
        ],
        [
            0,
            1,
            2
        ],
        [
            0,
            2,
            1
        ],
        [
            0,
            2,
            2
        ],
        [
            1,
            0,
            0
        ],
        [
            1,
            0,
            1
        ],
        [
            1,
            0,
            2
        ],
        [
            1,
            1,
            0
        ],
        [
            1,
            1,
            1
        ],
        [
            1,
            1,
            2
        ],
        [
            1,
            2,
            0
        ],
        [
            1,
            2,
            1
        ],
        [
            1,
            2,
            2
        ],
        [
            2,
            0,
            0
        ],
        [
            2,
            0,
            1
        ],
        [
            2,
            0,
            2
        ],
        [
            2,
            1,
            0
        ],
        [
            2,
            1,
            1
        ],
        [
            2,
            1,
            2
        ],
        [
            2,
            2,
            0
        ],
        [
            2,
            2,
            1
        ],
        [
            2,
            2,
            2
        ],
        [
            3,
            0,
            0
        ],
        [
            3,
            0,
            1
        ],
        [
            3,
            0,
            2
        ],
        [
            3,
            1,
            0
        ],
        [
            3,
            1,
            1
        ],
        [
            3,
            2,
            0
        ],
        [
            3,
            2,
            1
        ],
        [
            3,
            2,
            2
        ],
        [
            6,
            0,
            0
        ],
        [
            6,
            0,
            2
        ],
        [
            6,
            1,
            0
        ],
        [
            6,
            1,
            1
        ],
        [
            6,
            1,
            2
        ],
        [
            6,
            2,
            0
        ],
        [
            6,
            2,
            1
        ],
        [
            6,
            2,
            2
        ]
    ],
    "version": "0.1"
}